pkg/systemd/hwinfo.jsx

```diff
@@ -252,6 +252,7 @@ class HardwareInfo extends React.Component {
 
             pci = (
                 <ListingTable caption={ _("PCI") }
+                    rowKey={({ rowData, rowIndex }) => rowData.slot.title}
```

Comment:
If I understand this correctly, then `.slot` will change if the title is translated. And of course it's not entirely obvious what happens if you translate `_("Slot")` into a language which doesn't have lower case and such.. :sob: